src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -122,7 +591,7 @@
     "name": "Cache_Redis_SSL_Only",
     "description": "Only secure connections to your Azure Cache for Redis should be enabled",
     "recommendation": "Enable connections via SSL only to Redis Cache",
-    "helpUri": "https://github.com/Azure/template-analyzer/docs/built-in-bpa-rules.md/#only-secure-connections-to-your-azure-cache-for-redis-should-be-enabled",
+    "helpUri": "https://github.com/Azure/template-analyzer/blob/main/docs/built-in-bpa-rules.md/#onlysecureconnectionstoyourazurecacheforredisshouldbeenabled",
```

Comment:
missing dashes

Reply:
These are on purpose. For some reason those have the hyphens removed if you click on the anchor on the md page on GitHub.

Reply:
Strange...